.gitignore

```diff
@@ -1,3 +1,4 @@
-vendor
-composer.lock
-phpunit.xml
+/.idea
```

Comment:
That one should not be in the project gitignore, but in some local-only rule (or globally in your system). We don't want to maintain ignore rules for all IDEs that could be used when contributing: https://help.github.com/en/github/using-git/ignoring-files